### PR TITLE
Thread sphere fe 5

### PR DESCRIPTION
**Pull request by ellaycodes**

### CODE REVIEW

File: components/SingleArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { BodyHeader } from "./BodyHeader";
+import { getArticleById } from "../utils/api";
+import { useParams } from "react-router-dom";
+
+export const SingleArticle = () => {
+  const [article, setArticle] = useState([]);
+
```
Comment (nc-nick-jones) [inline components/SingleArticle.jsx]: we could do with some state in order to allow feedback for when the page is loading

File: components/AllArticles.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { BodyHeader } from "./BodyHeader"
```
Comment (nc-nick-jones) [inline components/AllArticles.jsx]: the casing of this import path does not match the name of your component.  this is causing errors when i try and launch the app.

File: components/SingleArticle.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id).then((article) => {
+      setArticle(article);
+    });
+  }, []);
+
+  console.log(article);
```
Comment (nc-nick-jones) [inline components/SingleArticle.jsx]: pls remove before commits

**Review by nc-nick-jones — CHANGES_REQUESTED** [review]
Hi Ella!  Nice job so far. Some feedback - 
Firstly, please could you move your components folder into your SRC folder? 

Also we need a loading pattern to give the user some feedback when loading up a new card.  

There are some issues with your imports that you might want to see to - see comments for details.  

Resubmit when you have a chance!  Mainly looking for that loading feedback ! 
You're doing well! Keep going :)

**Review by nc-ali-abdallah — APPROVED** [review]
awesome job implementing feedback.
Remember loading states are important to feedback to your user and you may need them in other tickets too!
Merge when ready!

**Comment by ellaycodes** [comment]
will pull into main

**Pull request closed without merging**